sdks/java/core/src/main/java/org/apache/beam/sdk/coders/CoderRegistry.java

```diff
@@ -250,30 +163,30 @@ public void registerCoder(Class<?> clazz, CoderFactory coderFactory) {
    *
    * @throws CannotProvideCoderException if there is no default Coder.
    */
-  public <InputT, OutputT> Coder<OutputT> getDefaultCoder(
+  public <InputT, OutputT> Coder<OutputT> getCoder(
       TypeDescriptor<OutputT> typeDescriptor,
       TypeDescriptor<InputT> inputTypeDescriptor,
       Coder<InputT> inputCoder)
       throws CannotProvideCoderException {
     checkArgument(typeDescriptor != null);
     checkArgument(inputTypeDescriptor != null);
     checkArgument(inputCoder != null);
-    return getDefaultCoder(
+    return createCoderFromTypeDescriptor(
```

Comment:
Use `getCoderFromTypeDescriptor` not `create`. As before, creation versus caching, etc, is an implementation detail.